lib/gooddata/models/process.rb

```diff
@@ -209,6 +209,48 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
         process
       end
 
+      def deploy_component(options = { :client => GoodData.client, :project => GoodData.project })
+        client, project = GoodData.get_client_and_project(options)
+
+        deploy_name = options[:name]
```

Comment:
Refers to 'options' more than self (maybe move it to another class?) (FeatureEnvy)